docs/redoc/master/openapi.json

```diff
@@ -7590,11 +7648,21 @@
         ]
       },
       "RecommendStrategy": {
-        "description": "How to use positive and negative examples to find the results, default is `average_vector`:\n\n* `average_vector` - Average positive and negative vectors and create a single query with the formula `query = avg_pos + avg_pos - avg_neg`. Then performs normal search.\n\n* `best_score` - Uses custom search objective. Each candidate is compared against all examples, its score is then chosen from the `max(max_pos_score, max_neg_score)`. If the `max_neg_score` is chosen then it is squared and negated, otherwise it is just the `max_pos_score`.",
-        "type": "string",
-        "enum": [
-          "average_vector",
-          "best_score"
+        "oneOf": [
```

Comment:
this change made python autogenerated structures look ugly

https://github.com/qdrant/qdrant-client/pull/678#discussion_r1670858885

cc: @generall